server/src/test/python/cromwell-az-e2e-test/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "cromwell-az-e2e-test"
+version = "0.1.0"
+description = ""
+authors = ["JVThomas <jthomas0209@gmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+requests = "^2.31.0"
```

Comment:
I really wish they would build `requests` into core Python, it's so often the only reason we need to worry about installing packages at all.